elasticsearch/deprecated.md

```diff
@@ -0,0 +1,3 @@
+This image is officially deprecated in favor of [the `elasticsearh` image provided by elastic.co](https://www.elastic.co/guide/en/elasticsearch/reference/5.2/docker.html) which is available to pull via `docker.elastic.co/elasticsearch/elasticsearch:[version]` like `5.2.1`. This image will receive no further updates after 2017-06-20 (June 20, 2017). Please adjust your usage accordingly.
+
+The team at Elastic.co prefer to offer Docker images on their infrastructure rather than through the "Official Images" program and the images are fully supported by them through their subscription program. You can also reach the community on the [forums](https://discuss.elastic.co/c/elasticsearch).
```

Comment:
We would prefer something like this, focusing strictly on non-commercial aspects:

Elastic provides open-source support for Elasticsearch via the [elastic/elasticsearch GitHub repository](https://github.com/elastic/elasticsearch) and the Docker image via the [elastic/elasticsearch-docker GitHub repository](https://github.com/elastic/elasticsearch-docker), as well as community support via its [forums](https://discuss.elastic.co/c/elasticsearch).